source/hooks/useInterval/index.js

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useRef } from 'react';
+
+const useInterval = (callback, delay) => {
```

Comment:
What do you think about linking something like https://overreacted.io/making-setinterval-declarative-with-react-hooks/ or any other source explaining why this is needed. It's not obvious at first.

Reply:
👍